README.md

```diff
@@ -123,7 +123,13 @@ The following table lists all valid connection properties:
 | PRIVATE_KEY_PWD            |No        | The passphrase to use for decrypting the private key, if the key is encrypted.|
 | PRIVATE_KEY                |Depends   | The private key to use for key-pair authentication. Must be used in combination with AUTHENTICATOR=snowflake_jwt. <br /> If the private key value includes any equal signs (=), make sure to replace each equal sign with two signs (==) to ensure that the connection string is parsed correctly.|
 | TOKEN                      |Depends   | The OAuth token to use for OAuth authentication. Must be used in combination with AUTHENTICATOR=oauth.|
-| INSECUREMODE               |No   	    | Set to true to disable the certificate revocation list check. Default is false.|
+| INSECUREMODE               |No   	| Set to true to disable the certificate revocation list check. Default is false.|
+| USEPROXY                   | No       | Set to true if you need to use a proxy server. The default value is false. <br/> <br/> This parameter was introduced in v2.0.4. |
+| PROXYHOST                  | No       | The hostname of the proxy server. <br/> <br/> This parameter was introduced in v2.0.4. |
+| PROXYPORT                  | No       | The port number of the proxy server. <br/> <br/> This parameter was introduced in v2.0.4. |
+| PROXYUSER                  | No       | The username for authenticating to the proxy server. <br/> <br/> This parameter was introduced in v2.0.4. |
+| PROXYPASSWORD              | No       | The password for authenticating to the proxy server. <br/> <br/> This parameter was introduced in v2.0.4. |
```

Comment:
Here also, this is required if USEPROXY is true and PROXYUSER is provided

Reply:
Thanks! Done